From: Jiri Slaby <jslaby@suse.cz>
Subject: kabi: x86/topology: Add CPUID.1F multi-die/package support
Patch-mainline: never, kabi
References: jsc#SLE-5454

This is a fix for patch:
patches.suse/x86-topology-Add-CPUID.1F-multi-die-package-support.patch

It added a member to struct cpuinfo_x86. Move it to the end along with
other kabi fixups.

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 arch/x86/include/asm/processor.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/x86/include/asm/processor.h
+++ b/arch/x86/include/asm/processor.h
@@ -128,7 +128,6 @@ struct cpuinfo_x86 {
 	u16			logical_proc_id;
 	/* Core id: */
 	u16			cpu_core_id;
-	u16			cpu_die_id;
 	u16			logical_die_id;
 	/* Index into per_cpu list: */
 	u16			cpu_index;
@@ -140,6 +139,7 @@ struct cpuinfo_x86 {
 	 * NOTE: only to be used for l1tf mitigation
 	 */
 	u8			x86_cache_bits;
+	u16			cpu_die_id;
 #endif
 };
 
